google_guest_agent/network/manager/dhclient_linux.go

```diff
@@ -56,21 +58,105 @@ var (
 	// ipv6 is a wrapper containing the protocol version and its respective
 	// dhclient argument.
 	ipv6 = ipVersion{"ipv6", "-6"}
-)
 
-var (
 	// baseDhclientDir points to the base directory for DHClient leases and PIDs.
 	baseDhclientDir = defaultBaseDhclientDir
+
+	// vlanIfaceCommonSet is a set of commands to setup common elements of a vlan interface
+	// it does sets link and dev level configurations.
```

Comment:
Nit: not sure if "does" should be included"

Reply:
done. thanks.